bundle/manifests/nginx-ingress-operator.clusterserviceversion.yaml

```diff
@@ -2,38 +2,33 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: >-
-      [{ "apiVersion": "k8s.nginx.org/v1alpha1", "kind": "NginxIngressController", "metadata": { "name": "my-nginx-ingress-controller", "namespace": "my-nginx-ingress" }, "spec": { "ingressClass": "nginx", "image": { "pullPolicy": "Always", "repository": "docker.io/nginx/nginx-ingress", "tag": "1.11.1-ubi" }, "nginxPlus": false, "serviceType": "NodePort", "type": "deployment" } }]
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "k8s.nginx.org/v1alpha1",
+          "kind": "NginxIngressController",
+          "metadata": {
+            "name": "nginxingresscontroller-sample"
+          },
+          "spec": {
+            "foo": "bar"
+          }
+        }
+      ]
     capabilities: Basic Install
-    categories: Networking
-    certified: "true"
-    containerImage: registry.connect.redhat.com/nginx/nginx-ingress-operator:0.2.0
-    createdAt: "2020-03-03T12:59:59Z"
-    description: The NGINX Ingress Operator is a Kubernetes/OpenShift component which
-      deploys and manages one or more NGINX/NGINX Plus Ingress Controllers
-    repository: https://github.com/nginxinc/nginx-ingress-operator
-    support: NGINX
-  name: nginx-ingress-operator.v0.2.0
+    operators.operatorframework.io/builder: operator-sdk-v1.8.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: nginx-ingress-operator.v0.3.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
     - description: NginxIngressController is the Schema for the nginxingresscontrollers
         API
+      displayName: Nginx Ingress Controller
       kind: NginxIngressController
       name: nginxingresscontrollers.k8s.nginx.org
-      displayName: NginxIngressController
```

Comment:
Do we need to add these back in manually again?

Reply:
Do you mean the old bundles? It looks like there's only one that gets overwritten https://sdk.operatorframework.io/docs/olm-integration/quickstart-bundle/#creating-a-bundle

Reply:
No I mean the removed lines `displayName: NginxIngressController` etc

Reply:
I think GitHub is showing a weird diff, like `dispalyName` is still there...not sure about `resources` I guess it's not needed anymore. There are a few that are OpenShift specific that I think we should address later all at once with the other OpenShift specific changes.

Reply:
the marker was in the wrong place, now everything (not Openshift specific) should be back